pkg/artifactory/resource_artifactory_single_replication_config_test.go

```diff
@@ -65,6 +65,10 @@ func testAccCheckSingleReplicationDestroy(id string) func(*terraform.State) erro
 
 		replica, resp, err := client.V1.Artifacts.GetRepositoryReplicationConfig(context.Background(), rs.Primary.ID)
 
+		if resp == nil {
```

Comment:
Check `err` first. If you check error first you probably never need to check if `resp` is `nil`.